extensions/binding/org.eclipse.smarthome.binding.sonos/src/main/java/org/eclipse/smarthome/binding/sonos/handler/ZonePlayerHandler.java

```diff
@@ -2621,11 +2556,6 @@ public String getCurrentZoneName() {
         return stateMap.get("CurrentZoneName");
     }
 
-    public String getCurrentURIFormatted() {
```

Comment:
Should this method not be kept in place (without the call to updateCurrentURIFormatted)?